src/main/java/com/minecolonies/coremod/colony/buildings/AbstractBuildingHut.java

```diff
@@ -33,6 +42,43 @@ public int getMaxInhabitants()
         return 1;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this method and declare a constant for this value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3400)